source/Ship.cpp

```diff
@@ -2042,10 +2042,16 @@ void Ship::DoGeneration()
 		// close to the star. Carried fighters can't collect fuel or energy this way.
 		if(currentSystem)
 		{
-			double scale = .2 + 1.8 / (.001 * position.Length() + 1);
-			fuel += currentSystem->SolarWind() * .03 * scale * (sqrt(attributes.Get("ramscoop")) + .05 * scale);
+			double ramscoop = max(attributes.Get("ramscoop"), .05);
+			double scale = 0.5 * pow(Mass(), 2. / 3.); // Reusing "scale" instead of making a new temporary variable.
 			
-			double solarScaling = currentSystem->SolarPower() * scale;
+			ramscoop = scale - (scale * scale) / (scale + ramscoop);
```

Comment:
equivalent to `scale * ramscoop / (scale + ramscoop)`.

Reply:
Oh, I hadn't realized! I thought this was the lowest, simplest form this could take, but apparently not! Tossing it into desmos, I can confirm they are equivalent.